<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QPushButton" name="mainButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-radius:10px;
	
	background-color: rgb(255, 255, 255);
}
QPushButton:pressed{
	padding-left:3px;
	padding-top:3px;
}


</string>
        </property>
        <property name="text">
         <string>主菜单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(233, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
	border:none;
	border-radius:10px;
}
QPushButton:pressed{
	padding-left:3px;
	padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>别点我x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="DrawFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>11</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
<<<<<<< HEAD
=======
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
>>>>>>> 75a5aef (fix:command)
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="MainMenu">
    <property name="font">
     <font>
      <family>华文隶书</family>
     </font>
    </property>
    <property name="title">
     <string>主菜单</string>
    </property>
    <addaction name="actionlook"/>
   </widget>
   <addaction name="MainMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlook"/>
  </widget>
  <action name="actionlook">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/img/game.jpg</normaloff>:/images/img/game.jpg</iconset>
   </property>
   <property name="text">
    <string>你瞅啥</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>outButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
